sample-compose/domain/src/main/java/co/nimblehq/sample/compose/domain/exceptions/Exceptions.kt

```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.sample.compose.domain.exceptions
+
+import co.nimblehq.sample.compose.domain.model.Error
+
+object NoConnectivityException : RuntimeException()
+
+data class ApiException(
+    val error: Error?,
```

Comment:
I have a question. 🙋🏻  While the Error model has only `message: String`, why don't we declare the error property in `ApiException` as just String?
```suggestion
    val errorMessage: String,
```

Reply:
@lydiasama As it's the template, the `Error` model could be more complex in real projects so we would like to keep the bare logic for future implementation,